src/main/java/io/zeebe/monitor/zeebe/kafka/ZeebeKafkaStreams.java

```diff
@@ -0,0 +1,85 @@
+package io.zeebe.monitor.zeebe.kafka;
```

Comment:
may be put this next to `ZeebeHazelcastService`?

Reply:
moved